database/migrations/postgres/000003_create_transactions.up.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE transactions (
-                              id BIGINT NOT NULL,
+                              id SERIAL PRIMARY KEY,
```

Comment:
Also here the original type of this column is BIGSERIAL 
https://github.com/bitcoin-sv/arc/blob/31aa8190dff210b87c12378adb7a67cc286aeb4e/blocktx/store/sql/sql.go#L200

Reply:
thanks, amended.